### PR TITLE
Change go repository. Expose IPV6 on "EXPOSE_IPV6" .Env or .Labels. Change version string.

### PR DESCRIPTION
**Pull request by ledgr**

### CODE REVIEW

File: bridge/util.go [hunk 1]
```diff
@@ -71,11 +75,20 @@ func servicePort(container *dockerapi.Container, port dockerapi.Port, published
 	} else {
 		ept = "tcp"  // default
 	}
+
+	metadata := serviceMetaData(container.Config, ep)
+	expose_ipv6 := mapDefault(metadata, "expose_ipv6", "false")
+	if expose_ipv6 == "false" {
```
Comment (ton31337) [inline bridge/util.go]: this is useless if, because it's default. set this as ipv4 and use only one check for ipv6 without any else statements.

**Comment by ton31337** [comment]
:+1:

File: bridge/util.go [hunk 2]
```diff
@@ -71,11 +75,20 @@ func servicePort(container *dockerapi.Container, port dockerapi.Port, published
 	} else {
 		ept = "tcp"  // default
 	}
+
+	metadata := serviceMetaData(container.Config, ep)
+	expose_ipv6 := mapDefault(metadata, "expose_ipv6", "false")
```
Comment (ton31337) [inline bridge/util.go]: is this really string, not bool?

File: bridge/util.go (hunk 1 above)
Comment (ledgr) [inline bridge/util.go]: `mapDefault()` will return any value set on docker container environment variable or label `EXPOSE_IPV6`.
If there is no Env or Label named `EXPOSE_IPV6` or value is empty, then `mapDefault()` will return `default` value, which in this case is "false".
So yes, this `metadata` map is a string map and so I'm doing string comparison here.

```go
func mapDefault(m map[string]string, key, default_ string) string {
        v, ok := m[key]
        if !ok || v == "" {
                return default_
        }
        return v
}
```

So you either have to **make sure** you set EXPOSE_IPV6 value to "true" and only to "true", or you can set it to any value but then function should look like (reverse of what is above):

```go
eip = container.NetworkSettings.GlobalIPv6Address
if expose_ipv6 == "false" { 
    eip = container.NetworkSettings.IPAddress
}
```

Comment (ton31337) [inline bridge/util.go]: In both ways if will be run :smile:

**Comment by ton31337** [comment]
Let's try :+1:

**Pull request merged**